code/extensions/che-remote/yarn.lock

```diff
@@ -298,13 +298,13 @@
     node-fetch "^2.6.0"
     url-parse "^1.4.3"
 
-"@eclipse-che/che-devworkspace-generator@7.86.0":
-  version "7.86.0"
-  resolved "https://registry.yarnpkg.com/@eclipse-che/che-devworkspace-generator/-/che-devworkspace-generator-7.86.0.tgz#6ee14f31e79a9c9b52979cda6e303b1b0429d54e"
-  integrity sha512-7FxGzvEkB/GbCQRgzdNpCWd2pIHhkzjpckRMKOIjpAroLFlnkRZSeoosPhSiRYKLNt2YAf2F1BXhqzUrswTuFQ==
+"@eclipse-che/che-devworkspace-generator@7.88.0-next-a2e5c63":
+  version "7.88.0-next-a2e5c63"
+  resolved "https://registry.yarnpkg.com/@eclipse-che/che-devworkspace-generator/-/che-devworkspace-generator-7.88.0-next-a2e5c63.tgz#2516aad6674fefeb9376974cf27855d573e2de36"
+  integrity sha512-70VbsUzONiyJr9pe/rslzta/SM4RvDsSR/kLIxMS4EbS9YgxEvT3rSM9B8eFLMbeZksnCuEQYxiyjSN6P4UV7A==
   dependencies:
-    "@devfile/api" "2.2.2-1700686170"
-    axios "1.6.0"
+    "@devfile/api" "2.2.2-1715367693"
```

Comment:
hmm.. why not 2.3.0 ?

Reply:
it's a transitive dependency
it means that `che-devworkspace-generator@7.88.0-next-a2e5c63` depends on `devfile/api" "2.2.2-1715367693`
so - the answer on the `che-devworkspace-generator` side

I'm not sure, but probably it should have been changed within https://github.com/eclipse-che/che-devfile-registry/pull/828
@vinokurig could you share your opinion?

Reply:
I checked that even `7.88.0-next-b9db1cf` version that was published 4 hours ago - brings `"@devfile/api" "2.2.2-1716821574"`, not `2.3.0`

<img width="757" alt="image" src="https://github.com/user-attachments/assets/5a159ce3-e88d-4426-b3bd-21dec969d3ac">

<img width="534" alt="image" src="https://github.com/user-attachments/assets/3f917a49-4fcb-4dbe-9b58-2b3f8618b24b">